epub33/core/index.html

```diff
@@ -6082,9 +6082,112 @@ <h4>URLs in the OCF Abstract Container</h4>
 					</div>
 
 					<p> In the <a>OCF Abstract Container</a>, when a file uses a URL string to reference another file in
-						the container, the string MUST be a <a data-cite="url#path-relative-scheme-less-url-string"
-							>path-relative-scheme-less-URL string</a>, optionally followed by <code>U+0023 (#)</code>
-						and a <a data-cite="url#url-fragment-string">URL-fragment string</a>. </p>
+						the container, the string SHOULD be a 
```

Comment:
It wasn't here that s/MUST/SHOULD/.
Here it should still be a MUST 😊

Reply:
Ok, I misunderstood, that is where you pointed at... so the algorithm is a SHOULD and not a MUST, right?

I must admit when I wrote the text I hesitated, and I then went for a MUST...